docker/Dockerfile.dev

```diff
@@ -38,6 +44,13 @@ RUN groupadd -g "${BUILD_GID}" dev_user && \
 # Now switch to the newly created user
 USER dev_user
 
+RUN (pipx install poetry && pipx ensurepath) || \
```

Comment:
if installation fails with `pipx` (ubuntu <=22), install in the original manner and add the user's local `site-packages` to the path